### DISCUSSION TITLE
Do people use standalone home-manager with plasma-manager?

### DISCUSSION THREAD

**Discussion by toast003** [nix-community/plasma-manager#124] [General]
I want to make an option that sets up plasma's keyboard layouts to be the same as the system's layouts (`services.xserver.xkb.layout` on NixOS) using the `osConfig` argument. Since that is not available when using standalone home-manager, I want to know if this is something I should try to implement.

**Comment by magnouvean** [comment]
Not that I use plasma-manager in home-manager only mode, but I think we should have support for stand-alone home-manager (as it is a legitimate way of using home-manager). I still think it could be OK to have some options only work on NixOS (with full system configuration) though as long as it's an option that can be toggled manually by the user.

**Comment by Asqiir** [comment]
I do use it in standalone hm on some managed devices
